src/commands/createNewProject/ProjectCreateStep/ScriptProjectCreateStep.ts

```diff
@@ -106,6 +106,9 @@ local.settings.json
 # TypeScript output
 dist
 out
+
+# Managed dependencies folder
+ManagedDependencies/
```

Comment:
Would this make sense to put in the `# Azure Functions artifacts` section above?

Reply:
We have changed the location where function app dependencies are installed to LocalApplicationData\AzureFunctions\FunctionAppName\ManagedDependencies, so we no longer need modify this file.